src/classes/TDTM_TriggerHandler.cls

```diff
@@ -87,13 +87,16 @@ public class TDTM_TriggerHandler {
         // Get the classes to run. Note that we are using the local object name, so it works for objects
         // from Cumulus both in the package and unmanaged dev orgs.
         List<SObject> classesToCall = dao.getclassesToCallForObject(describeObj.getLocalName(), thisAction);
-        UTIL_Debug.debug(LoggingLevel.WARN, '****Classes To Call for ' + describeObj.getLocalName() + ': ' + JSON.serializePretty(classesToCall));
+//        UTIL_Debug.debug(LoggingLevel.WARN, '****Classes To Call for ' + describeObj.getLocalName() + ': ' + JSON.serializePretty(classesToCall));
```

Comment:
I had left this behind because it seemed like someone added it for other debugging that is probably not required at this point, but might be useful at some future point. I can remove it though.